sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/FormRecognizerClient.java

```diff
@@ -225,7 +205,7 @@
     @ServiceMethod(returns = ReturnType.COLLECTION)
     public SyncPoller<OperationResult, List<FormPage>>
         beginRecognizeContent(InputStream form, long length, FormContentType formContentType) {
```

Comment:
Still debating if I should keep this API, as I think this parameter set would be more commonly used. And want to keep it simple for the users to pass in `FormContentType` without having to create options even if it isn't necessarily required.

@srnagar  thoughts?

Reply:
If user studies show that this is a more commonly used parameter set, then I think having this method will make it easier for the majority of the users compared to creating options.